spec/models/invoice_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Invoice, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Should we put a relationships test in here for the customer:invoice relationship? e.g. should belong_to ...

Reply:
Updated the spec file to have that relationship test